app/konnect/api/ksearch/query-syntax.md

```diff
@@ -0,0 +1,102 @@
+
+## Features
+
+**Global and Regional Access:** The KSearch API is available in global and regional locations with regional awareness. This ensures that entities returned are entities that are relevant to their geographical location, improving response times and conform to data residency expectations.
+
+**Comprehensive Response:** The responses of the search query are uniform and contain a fixed number of standard attributes: “id”, “type”, “labels/tags” and “name”. In addition, entity specific attributes are returned in a general attributes object.
+
+**Security and Accessibility** The KSearch API will only return entities that the user has permissions to access. If a user is able to retrieve the entity in the list endpoint, then the user will be able to see the entity in the search response.
+
+**Advanced Query Language**
+The KSearch API supports an advanced query language with the selectors, reserved characters and and logical operators.
+
+
+## Supported Entities
+- Users
+- Teams
+- System Accounts
+- Developers
+- Developer Teams
+- _More added soon._ (See `GET /search/type` for the latest entity types supported.)
+
+## Query Syntax
+
+### Selectors
+
+Selectors are used to define the criteria of the search. The table below summarizes the different selectors and their meanings:
+
+| Selector                  | Meaning                                          |
+|---------------------------|--------------------------------------------------|
+| `@{attr.sub_attr}:{value}`| "Fuzzy" match for an attribute and its value     |
+| `type:{entity_type}`      | Limits search results to a specific entity type  |
+| `labels.{name}:{value}`   | Exact match for a label's value                  |
+| `{value}`                 | "Fuzzy" match equivalent to `@*:{value}`         |
+| `{value}*`                | Matches any value starting with `{value}`        |
+
+### Reserved Characters
+
+Certain characters have special meanings in the query syntax:
+
+| Character | Meaning                                      |
+|-----------|----------------------------------------------|
+| `*`       | Wildcard character                           |
+| `@`       | Indicates "Fuzzy" matching for attributes    |
+| `""`      | Denotes an exact match, is case insensitive and includes spaces |
+
+### Logical Operators
+
+Logical operators are used to combine multiple criteria in a search query:
+
+| Operator                          | Function                           |
+|-----------------------------------|------------------------------------|
+| `AND`                             | Intersection of one or more fields |
+| `OR`                              | Union of one or more fields        |
+| `NOT`                             | Exclusion of a field               |
+
+By combining these selectors, reserved characters, and logical operators, users can construct complex and precise queries to effectively utilize the KSearch API.
+
+## Example Queries
```

Comment:
@haydenlam How would these be used as part of automation at a company? I'm guessing they would use the API to search for an entity to act on the results of that search, but what kind of action would companies want to take?

Reply:
I'm not sure there is a single straight forward answer, especially when it comes to search. The traditional forms of automation we think about in Konnect usually assumes the 🤖 knows exactly what needs to be done, rather than looking for something that is ambiguous which is what "search" is for.

For some very specific use cases:
- Search could be used to find entities that are "out of compliant". eg: find all routes that don't have a path that starts with `/api`.
- If a user needs to select from a list of available entities. eg: attaching a route to a service, search could be employed to search and identify which service to attach the route to.